apps/indexer/ponder.schema.ts

```diff
@@ -67,11 +67,16 @@ export const accountPower = onchainTable(
       .bigint("last_vote_timestamp")
       .default(BigInt(0))
       .notNull(),
+    firstVoteTimestamp: drizzle
+      .bigint("first_vote_timestamp")
+      .default(BigInt(0))
+      .notNull(),
   }),
   (table) => ({
     accountPowerAccountIdx: index().on(table.accountId),
     accountPowerDaoIdx: index().on(table.daoId),
     lastVoteTimestamp: index().on(table.lastVoteTimestamp),
+    firstVoteTimestamp: index().on(table.firstVoteTimestamp),
```

Comment:
Why do you need an index on this column? bare in mind that creating indexers isn't a free process on the database, it needs to go through all the lines and store them on a data structure that facilitates querying but also costs performance